distributed/ColdStartDistributedTraining.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# [Distributed] Pre-training and fine-tuning an LLM on CPU on AG News with ThirdAI's UDT\n",
+    "\n",
+    "This training script shows how to run data parallel pre-training of an LLM from scratch on the popular AG News Dataset (https://www.kaggle.com/datasets/amananandrai/ag-news-classification-dataset) using ThirdAI's Universal Deep Transformer (UDT)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Import thirdai and activate license"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install datasets\n",
+    "# !pip3 install thirdai --upgrade\n",
+    "!pip3 install ray\n",
+    "\n",
+    "import thirdai\n",
+    "# activation key here"
```

Comment:
We need to add an activation key here, which we must generate with Keygen. We may need a keygen developer account for that. Lemme ask Tharun to add me to one. Else, the test for demos would fail.

Reply:
I tried but it gives this error :
`RuntimeError: The license was found to be invalid: You must have a full license to perform this operation.`